Ginger/GingerCoreCommon/GeneralLib/IObservableList.cs

```diff
@@ -40,15 +41,15 @@ public interface IObservableList : ICollection, IList, INotifyCollectionChanged,
 
         bool AvoidLazyLoad { get; set; }
 
-        string StringData { get; set; }
+        LazyLoadListDetails LazyLoadDetails { get; set; }
 
         MemoryStream StringDataMS { get; set; }
 
         int DataLen { get; set; }
 
         List<object> ListItems { get; }
 
-        void DoLazyLoadItem(string s);
+        //void DoLazyLoadItem(string s);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)